app/templates/gulp/_styles.js

```diff
@@ -40,12 +40,15 @@ gulp.task('styles', function () {
     addRootSlash: false
   };
 
-  var indexFilter = $.filter(paths.src + '/app/index.<%= props.cssPreprocessor.extension %>');
+  var indexFilter = $.filter('index.<%= props.cssPreprocessor.extension %>');
```

Comment:
base directory of vynil files is `paths.src + '/app/'`, we don't need path in filter because globling is applied from base path only. (see https://github.com/wearefractal/vinyl/blob/master/README.md#optionsbase)